### ISSUE TITLE
GitHub webhook not restarting the server?

### ISSUE THREAD

**Issue by mmarchini** [nodejs/github-bot#279]
So I merged #275 today, but the bot was still behaving like before the PR. I just logged into the bot instance and noticed the bot server was running since Aug 9 0:06 UTC, the PR was merged at Aug 9 21:07 UTC. The repository folder was correctly updated, but the server didn't restart to apply the changes. Is the server supposed to restart when a new commit is merged, or does it restart once a day (or only manually)?

**Comment by phillipj** [comment]
Hmz, strange. It is supposed to restart as soon as commits lands to master, here is the relevant [github-webhook](https://www.npmjs.com/package/github-webhook) configuration in the build repo:

- [github-bot-deploy-webhook.service](https://github.com/nodejs/build/blob/master/ansible/roles/github-bot/templates/github-bot-deploy-webhook.service.j2) 
- [github-bot-deploy-webhook.json](https://github.com/nodejs/build/blob/master/ansible/roles/github-bot/templates/github-bot-deploy-webhook.json.j2)

I've noticed blipps of it not working from time to time, both for the github-bot and in nodejs.org which has a very similar github-webhook setup.

Looking at the webhook events log under this repo' Settings-page doesn't give any valuable hints either. The two most recent push events sent to the github-bot server succeeded:

![Screenshot 2020-08-10 at 10 40 04](https://user-images.githubusercontent.com/1231635/89765091-ee6a2d80-daf5-11ea-9a7d-6ff919620029.png)
 
_Bottom line;_ I don't know why it didn't restart. Let's keep an eye on it going forward and dig deeper if the issue persists?

**Comment by Trott** [comment]
I don't think we're seeing this problem anymore, so I'm going to close this. Pinging @richardlau in case he knows differently and wants to re-open.
